mep3_simulation/webots_data/controllers/opponent_robot/opponent_robot.py

```diff
@@ -0,0 +1,92 @@
+import random
+
+from controller import Supervisor
+# create the Robot instance.
+
+supervisor = Supervisor()
+
+# get the time step of the current world.
+timestep = int(supervisor.getBasicTimeStep())
+opponent_node = supervisor.getSelf()
+name_field = opponent_node.getField('name')
+opponent_field = opponent_node.getField('translation')
+name = name_field.getSFString()
+
+position = [0, 0, 0]
+
+positions = [(-1.34, 0.437, 10), (-1.12, 0.706, 10), (-0.761, 0.786, 20),
+             (-0.13, 0.792, 10), (-0.764, 0.53, 10), (-0.848, 0.297, 15),
+             (-0.713, 0.063, 10), (-0.603, -0.127, 15)]
+
+positions_1 = [(-1.4, 0.0238, 10), (-1.22, -0.257, 10), (-1.13, -0.646, 10),
+               (-0.738, -0.904, 15), (-0.565, -0.666, 20),
+               (-0.827, -0.412, 15)]
```

Comment:
```suggestion
POSITIONS_1 = [(-1.34, 0.437, 10), (-1.12, 0.706, 10), (-0.761, 0.786, 20),
             (-0.13, 0.792, 10), (-0.764, 0.53, 10), (-0.848, 0.297, 15),
             (-0.713, 0.063, 10), (-0.603, -0.127, 15)]

POSITIONS_2 = [(-1.4, 0.0238, 10), (-1.22, -0.257, 10), (-1.13, -0.646, 10),
               (-0.738, -0.904, 15), (-0.565, -0.666, 20),
               (-0.827, -0.412, 15)]
```

Konstante se pisu sa velikim slovima